From 0bfbb283202371ba79e6c15687456a6b983efb84 Mon Sep 17 00:00:00 2001
From: Yang Gang <yanggang@byosoft.com.cn>
Date: Fri, 3 Jan 2025 16:18:42 +0800
Subject: [PATCH 01/27] efi: Correct ECPT table GUID

Refer to UEFI specification 2.10:
  #define EFI_CONFORMANCE_PROFILES_TABLE_GUID \
  { 0x36122546, 0xf7e7, 0x4c8f, \
  { 0xbd, 0x9b, 0xeb, 0x85, 0x25, 0xb5, 0x0c, 0x0b }}

Signed-off-by: Yang Gang <yanggang@byosoft.com.cn>
Fixes: 6b92c1735205 ("efi: Create ECPT table")
Reviewed-by: Heinrich Schuchardt <heinrich.schuchardt@canonical.com>
---
 include/efi_api.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/efi_api.h b/include/efi_api.h
index f07d074f93b..d3cf012693f 100644
--- a/include/efi_api.h
+++ b/include/efi_api.h
@@ -227,7 +227,7 @@ enum efi_reset_type {
 		 0x71, 0x94, 0x19, 0x9a, 0xd9, 0x2a)
 
 #define EFI_CONFORMANCE_PROFILES_TABLE_GUID \
-	EFI_GUID(0x36122546, 0xf7ef, 0x4c8f, 0xbd, 0x9b, \
+	EFI_GUID(0x36122546, 0xf7e7, 0x4c8f, 0xbd, 0x9b, \
 		 0xeb, 0x85, 0x25, 0xb5, 0x0c, 0x0b)
 
 #define EFI_CONFORMANCE_PROFILES_TABLE_VERSION 1
-- 
2.43.0

